src/main/java/beam/playground/beamSimAkkaProtoType/beamPersonAgent/BeamPersonAgent.java

```diff
@@ -61,10 +60,6 @@ public void onReceive(TriggerMessage message) {
             // new PersonArrivalAtVehicleMessage(getSelf(), arrivalTime, 0));
 
         } else if (message instanceof PersonArrivalAtVehicleMessage) {
-            if (vehicleIsPluggedIn) {
```

Comment:
@colinsheppard - this seems a bit EV-ish, so you may want to verify whether this should be removed, or fixed?